lib/node_modules/@stdlib/array/slice/docs/types/index.d.ts

```diff
@@ -39,274 +39,14 @@ import { Collection, Complex128Array, Complex64Array } from '@stdlib/types/array
-*
-* var out = slice( x, 0, 2 );
-* // returns <Uint16Array>[ 1, 2 ]
-*/
-declare function slice( x: Uint16Array, start?: number, end?: number ): Uint16Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint8Array = require( '@stdlib/array/uint8' );
-*
-* var x = new Uint8Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x );
-* // returns <Uint8Array>[ 1, 2, 3 ]
-*
-* @example
-* var Uint8Array = require( '@stdlib/array/uint8' );
-*
-* var x = new Uint8Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 2 );
-* // returns <Uint8Array>[ 1, 2 ]
-*/
-declare function slice( x: Uint8Array, start?: number, end?: number ): Uint8Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint8ClampedArray = require( '@stdlib/array/uint8c' );
-*
-* var x = new Uint8ClampedArray( [ 1, 2, 3 ] );
-*
-* var out = slice( x );
-* // returns <Uint8ClampedArray>[ 1, 2, 3 ]
-*
-* @example
-* var Uint8ClampedArray = require( '@stdlib/array/uint8c' );
-*
-* var x = new Uint8ClampedArray( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 2 );
-* // returns <Uint8ClampedArray>[ 1, 2 ]
-*/
-declare function slice( x: Uint8ClampedArray, start?: number, end?: number ): Uint8ClampedArray;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Complex128Array = require( '@stdlib/array/complex128' );
-*
-* var x = new Complex128Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
-*
-* var out = slice( x );
-* // returns <Complex128Array>
-*
-* @example
-* var Complex128Array = require( '@stdlib/array/complex128' );
-*
-* var x = new Complex128Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
-*
-* var out = slice( x, 0, 2 );
-* // returns <Complex128Array>
-*/
-declare function slice( x: Complex128Array, start?: number, end?: number ): Complex128Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
 * var Complex64Array = require( '@stdlib/array/complex64' );
 *
-* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
 *
 * var out = slice( x );
-* // returns <Complex64Array>
-*
-* @example
-* var Complex64Array = require( '@stdlib/array/complex64' );
-*
-* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
-*
-* var out = slice( x, 0, 2 );
-* // returns <Complex64Array>
+* // returns <Complex64Array>[ 1.0, 2.0, 3.0 ]
```

Comment:
This example is incorrect and should be reverted to the original. Complex number array buffers cannot have an odd number of elements.

Reply:
Fixed.